src/main/java/com/ericsson/ei/subscriptionhandler/SpringRestTemplate.java

```diff
@@ -73,13 +75,19 @@ public int postDataMultiValue(String notificationMeta, MultiValueMap<String, Str
                         headers);
                 response = rest.postForEntity(notificationMeta, request, JsonNode.class);
             }
+        } catch (HttpClientErrorException e) {
+            LOGGER.error("HTTP-request failed, bad request!\n When trying to connect to URL: " + notificationMeta);
+            return HttpStatus.BAD_REQUEST.value();            
+        } catch (HttpServerErrorException e) {
+            LOGGER.error("HTTP-request failed, internal server error!\n When trying to connect to URL: " + notificationMeta);
```

Comment:
Include e.getMessage() in Logger.error message. This apples for both these two catches.